src/MosparoIntegration/Helper/ModuleHelper.php

```diff
@@ -39,31 +42,33 @@ public static function getInstance()
 
     private function __construct()
     {
-
     }
 
     public function getActiveModules()
     {
         return $this->activeModules;
     }
 
-    public function getActiveModule($moduleKey)
+    public function getActiveModule($moduleKey) : ?AbstractModule
     {
         if (!isset($this->activeModules[$moduleKey])) {
             return null;
         }
-
         return $this->activeModules[$moduleKey];
     }
 
     public function initializeActiveModules($pluginDirectoryPath, $pluginDirectoryUrl)
     {
         $configHelper = ConfigHelper::getInstance();
+
         foreach (self::getAvailableModules() as $moduleClass) {
             $module = new $moduleClass();
-
             if ($configHelper->isModuleActive($module->getKey())) {
-                $module->initializeModule($pluginDirectoryPath, $pluginDirectoryUrl);
+                $configHelper->loadModuleConfiguration($module);
+                //Disable fields display and form validations for managers/admin users
+                if (!current_user_can('edit_users')) {
```

Comment:
I think we should always initialize the module (if it is enabled) and decide in the display and verification actions if we have to show and verify the request or if we can just pass it. This is - as far as I know - only relevant for the Lost password functionality since in all other cases, a logged in user could still submit spam.